### PR TITLE
Add weekly sync meeting notes 2021-09-27

### PR DESCRIPTION
**Pull request by vmx**

### CODE REVIEW

File: meeting-notes/2021/2021-09-27--ipld-sync.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  - Start live stream
+  - Find a notetaker
+  - Ask everyone to put their name into the list of attendees
+  - Ask everyone to put their items that they've been working on the past two weeks (should be done prior to the meeting)
+  - Ask for last minute agenda items
+- This meeting
+  - consider renaming+forwarding ipld-prime/multicodec to avoid confusing conflict with go-multicodec
+  - now that we deleted prime's schema2 and quip... can we remove dagjson2?
+  - _add your agenda item_
+  - to go-jst
```
Comment (mvdan) [inline meeting-notes/2021/2021-09-27--ipld-sync.md:27]: Is this line misplaced?

**Review by mvdan — APPROVED** [review]

File: meeting-notes/2021/2021-09-27--ipld-sync.md (hunk 1 above)
Comment (rvagg) [inline meeting-notes/2021/2021-09-27--ipld-sync.md:27]:
```suggestion
  - removing old code from go-ipld-prime
```

File: meeting-notes/2021/2021-09-27--ipld-sync.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+<!-- After each call, the notetaker submits a PR to https://github.com/ipld/team-mgmt to store the notes on the meeting-notes folder -->
+
+@mvdan: consider renaming+forwarding ipld-prime/multicodec to avoid confusing conflict with go-multicodec
+
+ - Renaming things to make things easier to understand
+
+@mvdan: now that we deleted prime's schema2 and quip... can we remove dagjson2?
+
+ - Yes.
```
Comment (rvagg) [inline meeting-notes/2021/2021-09-27--ipld-sync.md:89]:
```suggestion
@warpfork: Yes.
 
 @rvagg: and https://github.com/ipld/go-ipld-prime/tree/master/codec/jst too?
 
 @warpfork: probably, yes, the useful pieces have already been extracted to another module
```

**Comment by vmx** [comment]
Thanks @rvagg for taking time to make those changes!

**Pull request merged**